bucket/v2ray.json

```diff
@@ -15,23 +15,19 @@
     },
     "installer": {
         "script": [
-            "#v2ctl.exe cannot recognize signature file when there are spaces in path. This is a workaround to eliminate spaces.",
+            "#v2ray.exe cannot recognize signature file when there are spaces in path. This is a workaround to eliminate spaces.",
             "Write-Host 'Verifying V2Ray binary...' -f Yellow",
-            "$sig_dir = \"$Env:SystemRoot\\Temp\\v2ray\"",
+            "$sig_dir = \"$Env:TEMP\\v2ray\"",
```

Comment:
Was the issue with spaces in path fixed? Because $Env:TEMP can still contain space if the user's name has space.

Reply:
sorry i didn't notice this.